xbmc/utils/Splash.cpp

```diff
@@ -39,14 +45,46 @@ CSplash& CSplash::GetInstance()
   return instance;
 }
 
+void CSplash::Start(std::string txt /* = "" */)
+{
+  text = txt;
+  if (IsRunning())
+    return;
+  Create();
+}
+
+void CSplash::Stop(bool bWait /* = false */)
+{
+  if (!IsRunning())
+    return;
+
+  StopThread(bWait);
+}
+
+void CSplash::Process()
+{
+  int i = 1;
+  int max = 3;
+  while (!m_bStop)
+  {
+    for (int i = 1; i <= 3 && !m_bStop; ++i)
+    {
+      Show(std::string(i, ' ') + text + std::string(i, '.'));
+      if (!m_bStop)
+        Sleep(1000);
+    }
+  }
+  Show();
```

Comment:
As I said it's out of my area and I have n idea where to start so starting from scratch is probably too hard for me.. I'll close the pr so if someone wants to try it can..